# Set Debian configuration defaults.
--- shorewall.git.orig/configfiles/shorewall6.conf
+++ shorewall.git/configfiles/shorewall6.conf
@@ -10,7 +10,7 @@
 #		       S T A R T U P   E N A B L E D
 ###############################################################################
 
-STARTUP_ENABLED=No
+STARTUP_ENABLED=Yes
 
 ###############################################################################
 #		              V E R B O S I T Y
@@ -70,7 +70,7 @@
 
 SHOREWALL_SHELL=/bin/sh
 
-SUBSYSLOCK=/var/lock/subsys/shorewall
+SUBSYSLOCK=""
 
 TC=
 
