# Explicitely locate lock file.
--- shorewall.git.orig/lib.base
+++ shorewall.git/lib.base
@@ -104,7 +104,8 @@
     local try
     try=0
     local lockf
-    lockf=${LOCKFILE:=${VARDIR}/lock}
+    # lockf=${LOCKFILE:=${VARDIR}/lock}
+    lockf=/var/lock/shorewall6
     local lockpid
 
     MUTEX_TIMEOUT=${MUTEX_TIMEOUT:-60}
@@ -150,7 +151,8 @@
 #
 mutex_off()
 {
-    rm -f ${LOCKFILE:=${VARDIR}/lock}
+    # rm -f ${LOCKFILE:=${VARDIR}/lock}
+    rm -f /var/lock/shorewall6
 }
 
 #
