# Explicitely locate lock file.
--- shorewall-debian.orig/lib.base
+++ shorewall-debian/lib.base
@@ -105,7 +105,8 @@
     local try
     try=0
     local lockf
-    lockf=${LOCKFILE:=${VARDIR}/lock}
+    # lockf=${LOCKFILE:=${VARDIR}/lock}
+    lockf=/var/lock/shorewall6
 
     MUTEX_TIMEOUT=${MUTEX_TIMEOUT:-60}
 
@@ -136,7 +137,8 @@
 #
 mutex_off()
 {
-    rm -f ${LOCKFILE:=${VARDIR}/lock}
+    # rm -f ${LOCKFILE:=${VARDIR}/lock}
+    rm -f /var/lock/shorewall6
 }
 
 #
