# Explicitely locate lock file.
--- shorewall6-4.2.6.orig/lib.base
+++ shorewall6-4.2.6/lib.base
@@ -336,7 +336,8 @@
     local try
     try=0
     local lockf
-    lockf=${LOCKFILE:=${VARDIR}/lock}
+    # lockf=${LOCKFILE:=${VARDIR}/lock}
+    lockf=/var/lock/shorewall6
 
     MUTEX_TIMEOUT=${MUTEX_TIMEOUT:-60}
 
@@ -367,7 +368,8 @@
 #
 mutex_off()
 {
-    rm -f ${LOCKFILE:=${VARDIR}/lock}
+    # rm -f ${LOCKFILE:=${VARDIR}/lock}
+    rm -f /var/lock/shorewall6
 }
 
 #
